app/javascript/controllers/collection_participant_form_controller.js

```diff
@@ -1,4 +1,5 @@
 import { Controller } from "stimulus"
+import debounce from 'lodash.debounce'
```

Comment:
Should this be added to `package.json`?

Reply:
It worke fine for me on localhost without explicitly adding it, but I just added it now in case we need to deploy.

Reply:
Yeah, it is already included transitively. This just makes it explicit.